www/scripts/productlist/ListOfProducts.js

```diff
@@ -0,0 +1,220 @@
+// -------------------------------------------------------------------------
+//                     The CodeChecker Infrastructure
+//   This file is distributed under the University of Illinois Open Source
+//   License. See LICENSE.TXT for details.
+// -------------------------------------------------------------------------
+
+define([
+  'dojo/_base/declare',
+  'dojo/dom-construct',
+  'dojo/data/ItemFileWriteStore',
+  'dojo/topic',
+  'dijit/form/Button',
+  'dijit/form/TextBox',
+  'dijit/layout/BorderContainer',
+  'dijit/layout/ContentPane',
+  'dojox/grid/DataGrid',
+  'codechecker/util'],
+function (declare, domConstruct, ItemFileWriteStore, topic, Button,
+  TextBox, BorderContainer, ContentPane, DataGrid, util) {
+
+  function prettifyDuration(seconds) {
```

Comment:
This file contain the same function: https://github.com/whisperity/CodeChecker/blob/product_management/www/scripts/codecheckerviewer/ListOfRuns.js#L22.
Move it to a common place such as util.js.